JetAnalyzers/bin/jet_response_fitter_x.cc

```diff
@@ -36,6 +40,16 @@ using namespace std;
 // define local functions
 ////////////////////////////////////////////////////////////////////////////////
 
+/// set new histogram axis range to compute KS test
+TH1F *hXaxis( TH1F* hist, double minX, double maxX);
+
+/// optional modify crystal ball fit to response distribution
+void fit_modified_fnc_dscb(TH1F*& hrsp, string alg, bool calibrated, string histName); // used to call the global fit
+double sumGausPol7(double *x, double *par); // gaus+polynome 7 function
+double exp_fcn(double x, double norm, double slope); // exp function
+double gauss_fcn(double x, double norm, double mean, double sigma); // gauss function
+double pol7_fcn(double x, double p0, double p1, double p2, double p3, double p4, double p5, double p6, double p7); // poly 7 function
+double modified_fnc_dscb(double*xx,double*pp); // global fit using the function above
```

Comment:
Can you then comment each function like fit_gaussian? See below for example:

/// comment 1
void function1();

///comment 2
void function2();